tensorflow_addons/image/__init__.py

```diff
@@ -17,16 +17,17 @@
 from __future__ import division
 from __future__ import print_function
 
+from tensorflow_addons.image.connected_components import connected_components
 from tensorflow_addons.image.dense_image_warp import dense_image_warp
 from tensorflow_addons.image.dense_image_warp import interpolate_bilinear
 from tensorflow_addons.image.distance_transform import euclidean_dist_transform
 from tensorflow_addons.image.distort_image_ops import adjust_hsv_in_yiq
 from tensorflow_addons.image.distort_image_ops import random_hsv_in_yiq
 from tensorflow_addons.image.filters import mean_filter2d
 from tensorflow_addons.image.filters import median_filter2d
+from tensorflow_addons.image.interpolate_spline import interpolate_spline
```

Comment:
Because the import order is changed, you have to modify [this line](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/image/sparse_image_warp.py#L190) from
```python
interpolate_spline.interpolate_spline(
```
to
```python
interpolate_spline(
```

Reply:
Thanks a lot @WindQAQ.